filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -413,6 +466,89 @@ function createSubjectNode($doc, $subjectSchemeId, $subjectHeadingOrCode, $subje
 		return $subjectNode;
 	}
 
+	/**
+	 * Append the collection node 'collection property="crawler-based"' to the Article/Issue data node.
+	 * @param $doc DOMDocument
+	 * @param $deployment Deployment
+	 * @param $request Request
+	 * @param $context Context
+	 * @param $crawlableDataNode DOMElement
+	 * @param $article Article/Issue
+	 * @param $galleys array of galleys
+	 */
+	function appendAsCrawledCollectionNodes($doc, $deployment, $request, $context, $crawlableDataNode, $article, $galleys) {
+
+		foreach ($galleys as $crawledGalley) {
+			// loop through values 
+			// as-crawled URL - collection nodes
+		   $urlPath_ = array($article->getBestArticleId(), $crawledGalley->getBestGalleyId());
+		   $resourceURL_ = $request->url($context->getPath(), 'article', 'view', $urlPath_, null, null, true);
```

Comment:
I think, this should be the file download URL i.e. 'download' instead of 'view'.